Build fails due to missing kustomize

**Describe the bug**
Following the README to setup my first operator, `make install deploy` fails with:

    [...snip...]
    kustomize build config/crd > helm/solr-operator/crds/crds.yaml
    /bin/sh: kustomize: command not found
    make: *** [manifests] Error 127

**To Reproduce**
Steps to reproduce the behavior:
1. `make install deploy` on a clean Mac system with Docker Desktop and Go installed
2. See above error

**Expected behavior**
Build succeeds

**Environment (please complete the following information):**
 - macOS 10.15.3
 - Docker Desktop 2.2.0.4 with Kubernetes
 - Go 1.14 installed via brew

**Additional context**
Add any other context about the problem here.

--- a/README.md
+++ b/README.md
@@ -40,6 +40,12 @@ Each is optional, as described in the [Zookeeper](#zookeeper-reference) section.
 $ kubectl apply -f example/dependencies
 ```
 
+Install necessary dependencies for building and deploying the operator.
+```bash
+$ export PATH="$PATH:$GOPATH/bin" # You likely want to add this line to your ~/.bashrc or ~/.bash_aliases
+$ ./hack/install_dependencies.sh
+```
+
 Install the Solr CRDs & Operator
 
 ```bash

